packages/forms/frontend/sirius-components-widget-reference/src/ReferencePreview.tsx

```diff
@@ -28,11 +28,12 @@ import { GQLReferenceWidgetStyle } from './ReferenceWidgetFragment.types';
 
 const useStyles = makeStyles<Theme, GQLReferenceWidgetStyle>((theme) => ({
   style: {
-    color: ({ color }) => (color ? getCSSColor(color, theme) : null),
-    fontSize: ({ fontSize }) => (fontSize ? fontSize : null),
-    fontStyle: ({ italic }) => (italic ? 'italic' : null),
-    fontWeight: ({ bold }) => (bold ? 'bold' : null),
-    textDecorationLine: ({ underline, strikeThrough }) => getTextDecorationLineValue(underline, strikeThrough),
+    color: ({ color }) => (color ? getCSSColor(color, theme) : undefined),
+    fontSize: ({ fontSize }) => (fontSize ? fontSize : undefined),
+    fontStyle: ({ italic }) => (italic ? 'italic' : 'unset'),
+    fontWeight: ({ bold }) => (bold ? 'bold' : 'unset'),
```

Comment:
How could you tell that the default value should be `unset`? Didn't we conclude that `null` was used to do nothing at all? Otherwise create the style using a real piece of code.

Reply:
`null` is not a valid value for `FontStyleProperty`, with strict mode it raises an error